<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>646</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="init_Button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Find devices</string>
    </property>
   </widget>
   <widget class="QPushButton" name="update_Button_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Update</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>10</y>
      <width>181</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="connect_Button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>10</y>
      <width>311</width>
      <height>111</height>
     </rect>
    </property>
    <property name="tabStopDistance">
     <double>20.000000000000000</double>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
    </property>
   </widget>
   <widget class="QPushButton" name="singleMeasure_Button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Single measure</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Configure
detector</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="data_plainTextEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>281</width>
      <height>301</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>151</y>
      <width>188</width>
      <height>120</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Integration time</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSpinBox" name="intergationTime_spinBox">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>100000</number>
       </property>
       <property name="value">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>ms</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Averages</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="averages_spinBox">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>1000</number>
       </property>
       <property name="value">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Measurments</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="measurments_spinBox">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Integration delay</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QSpinBox" name="integrationDelay_spinBox">
       <property name="maximum">
        <number>1000000</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="LED_pushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>LED</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="ledOff_measure_pushButton">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>150</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>LED off and
single measure</string>
    </property>
   </widget>
   <widget class="QPushButton" name="program1_pushButton">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>200</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Program-1</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>150</y>
      <width>171</width>
      <height>27</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>LED off delay</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="ledOff_delay_spinBox"/>
     </item>
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>ms</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="program2_pushButton">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>230</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Program-2</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
